guides/source/action_cable_overview.md

```diff
@@ -178,9 +178,6 @@ established using the following JavaScript, which is generated by default by Rai
 
 ```js
 // app/javascript/channels/consumer.js
-// Action Cable provides the framework to deal with WebSockets in Rails.
-// You can generate new channels where WebSocket features live using the `rails generate channel` command.
```

Comment:
Have you considered mentioning the `rails generate channel` command outside of the code example? Otherwise I think it you'd be removing it from the guide entirely.

Reply:
No, I didn't consider it. Thanks for mentioning it.
Most other example of `rails generate` I see in the guides are in 'Getting Started'